ngraph/python/tox.ini

```diff
@@ -15,15 +15,17 @@ setenv =
   LD_LIBRARY_PATH = {env:LD_LIBRARY_PATH:{homedir}/ngraph_dist/lib}
   DYLD_LIBRARY_PATH = {env:DYLD_LIBRARY_PATH:{homedir}/ngraph_dist/lib}
   PYBIND_HEADERS_PATH = {env:PYBIND_HEADERS_PATH:}
-  NGRAPH_BACKEND = {env:NGRAPH_BACKEND:"INTERPRETER"}
+  NGRAPH_BACKEND = {env:NGRAPH_BACKEND:"CPU"}
```

Comment:
It would be great if we could still test the reference implementations of nGraph in the case of ONNX library.

Reply:
Agreed. Not available at this point, unfortunately. We can add it later.